engine/storage/volume/src/main/java/org/apache/cloudstack/storage/datastore/provider/DefaultHostListener.java

```diff
@@ -121,7 +122,9 @@ private NicTO createNicTOFromNetworkAndOffering(NetworkVO networkVO, NetworkOffe
     public boolean hostConnect(long hostId, long poolId) throws StorageConflictException {
         StoragePool pool = (StoragePool) this.dataStoreMgr.getDataStore(poolId, DataStoreRole.Primary);
         ModifyStoragePoolCommand cmd = new ModifyStoragePoolCommand(true, pool);
-        cmd.setWait(60);
+        cmd.setWait(Wait.value() / 5);
```

Comment:
Why divide by 5?

Reply:
For ModifyStoragePoolCommand, we don't externalize the timeout to avoid confusion for the user. Since, the required timeout can vary depending on the provider in use and we are only setting the wait for default host listener for now. Instead, we reuse the global wait setting by dividing it by 5 making the default value of 6 minutes (1800/5 = 360s) for ModifyStoragePoolCommand.

Reply:
It still does not make sense. Why 5? Just to match 6 minutes when the default `wait` is 1800?

Reply:
Yes. Reasoning behind this is that user might decrease the wait in some scenario. It's quite unlikely but if the user sets it to 5 min, we will wait at least 1 minute here.

P.S. - the actual wait time is twice the value set for wait. Check the note in the description.

Reply:
User also might increase `wait` to 18000 and we would have 1 hour here. It does not make sense to use an unfounded math function to derive a value; please, either use the same configuration or externalize a new one.

Reply:
It's more about the worst case scenario. It will wait for the specified time only if the agent is stuck because of some reason. Before #8502, if the wait was set to 18000, this command would have waited for 10 hours (5 hours *2). This solution doesn't completely solve the the problem but reduces the issues to some extent until issues #8506 or #8011 is resolved.

Reply:
In this case, externalizing a new configuration is the best choice.

Reply:
In contrast with taking a value and dividing it by a random number, externalizing a new configuration would be the best option. We will not externalize every command timeout right now, so it would not be a problem. I agree a more robust solution, like what I mentioned in https://github.com/apache/cloudstack/pull/8502#discussion_r1450506118, is better; however, we have to discuss it first.

As last resort, if you do not want to externalize a new configuration, creating a constant with javadoc for this case is better than deriving the value.